tools/release-post-generator/lib/environment.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Internal dependencies
+ */
+import { Logger } from './logger';
+
+export const getEnvVar = ( varName: string, isRequired = false ) => {
```

Comment:
basic access to env vars set by `dotenv`. The idea is to stop the program if a env var is required and not provided, this should help people realize what is wrong when their command doesn't work.

I'd like to reuse this pattern in our other tools as we move them to Commander